vector/src/main/java/im/vector/adapters/VectorMessagesAdapterHelper.java

```diff
@@ -552,6 +552,45 @@ public void onClick(View view) {
         }
     }
 
+    /** Determine how many backticks bracket a message, for code block
+     * handling (issue 145) */
+    public int getTickCount(final Message message) {
+        final String mb = message.body;
+        if (mb.startsWith("```") && mb.endsWith("```")) {
+            return 3;
+        }
+        else if (mb.startsWith("``") && mb.endsWith("``")) {
+            return 2;
+        }
+        else if (mb.startsWith("`") && mb.endsWith("`")) {
+            return 1;
+        }
+        else {
+            return 0;
```

Comment:
give a name for ``, ``, ` block

Reply:
In my implementation of your review, this function no longer exists.